src/pages/index.js

```diff
@@ -137,18 +205,71 @@ function closeModal(modal) {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
-  closeModal(editModal);
+  const submitBtnTwo = evt.submitter;
+  setButtonText(submitBtnTwo, true, "Saving...", "Save");
+  //submitBtnTwo.textContent = "Saving...";
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      profileAvatar.src = data.avatar;
+      evt.target.reset();
+      disableButton(submitBtn, config.inactiveButtonClass);
+      closeModal(editModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(submitBtnTwo, false, "Saving...", "Save");
```

Comment:
`"Saving...", "Save"`


It should be `"Save", "Saving..."` because the 3rd argument is `defaultText`, and the 4th is `loadingText`


```js
  setButtonText(submitBtnTwo, false, "Save", "Saving...");


```